java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -183,4 +183,73 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Long> zcard(String key);
+
+    /**
+     * Returns a random element from the sorted set stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/zrandmember/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @return A <code>String</code> representing a random element from the sorted set.<br>
+     *     If the sorted set does not exist or is empty, the response will be <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * String random1 = client.zrandmember("mySortedSet").get();
+     * assert random1.equals("GLIDE");
+     *
+     * String random2 = client.zrandmember("nonExistingSortedSet").get();
+     * assert random2 == null;
+     * }</pre>
+     */
+    CompletableFuture<String> zrandmember(String key);
+
+    /**
+     * Retrieves random elements from the sorted set stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/zrandmember/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param count The number of elements to return<br>
```

Comment:
```suggestion
     * @param count The number of elements to return:<br>
```
colon or period is missing